onnxruntime/core/providers/coreml/model/model.mm

```diff
@@ -313,8 +368,8 @@ - (Status)predict:(const std::unordered_map<std::string, OnnxTensorData>&)inputs
       const auto static_output_shape = GetStaticOutputShape(output_tensor_info.shape, coreml_static_output_shape,
                                                             *logger_);
 
-      void* output_buffer = get_output_tensor_mutable_raw_data_fn(output_name, output_tensor_info.data_type,
-                                                                  static_output_shape);
+      __block void* output_buffer = get_output_tensor_mutable_raw_data_fn(output_name, output_tensor_info.data_type,
```

Comment:
does this variable need to be declared with `__block`? the pointer itself is not modified in the block.